test/test_assets_d8/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+gem 'sass', '~> 3.4.9'
+gem 'compass', '~> 1.0.1'
```

Comment:
Adding a json library makes this a bit faster and is a best practice we've got documented: https://github.com/phase2/grunt-drupal-tasks/blob/master/CONFIG.md#setting-up-gem-dependencies